File: mRemoteV1/CS/Connection/Connection.Protocol.Serial.cs
using System.Collections.Generic;
using System;
using AxWFICALib;
using System.Drawing;
using System.Diagnostics;
using System.Data;
using AxMSTSCLib;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;


namespace mRemoteNG.Connection.Protocol
{
	public class Serial : PuttyBase
	{
				
		public Serial()
		{
			this.PuttyProtocol = Putty_Protocol.serial;
		}
				
		public enum Defaults
		{
			Port = 9600
		}
	}
}

File: mRemoteV1/Connection/Connection.Protocol.Serial.vb
Namespace Connection

    Namespace Protocol
        Public Class Serial
            Inherits PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.serial
            End Sub

            Public Enum Defaults
                None = 0
                Port = 9600
            End Enum
        End Class
    End Namespace

End Namespace